gradle/libs.versions.toml

```diff
@@ -1,13 +1,13 @@
 [versions]
-opendcs = "7.0.10"
+opendcs = "7.0.11"
 
 jetty = "9.4.53.v20231009"
-servlet-api = "3.1.0"
-slf4j = "2.0.0"
+servlet-api = "4.0.1"
```

Comment:
This shouldn't be updated arbitrarily. certain versions require jakarta.* namespace and java 11+

Is there a reason this was updated?

Reply:
Good point, I will revert this.  I left this pull request in draft because I knew there were some items that needed some cleaning up..  We were just doing it through intellij so I could get the hang of doing it through intellij.  I will clean it up and when it's ready make it a non-draft.

Reply:
4.0.1 was the final release before migrating to jakarta. The maven artifact shows it was compiled with JDK 8: https://mvnrepository.com/artifact/javax.servlet/javax.servlet-api/4.0.1